.github/workflows/backcompat.yml

```diff
@@ -1,36 +0,0 @@
-name: Backwards Compatability
```

Comment:
which workflow is triggering those tests now ?

Reply:
They are included in the regular build. The compile for the tests is compiling these.